### PR TITLE
Long-term contribution documentation solutions

### PR DESCRIPTION
**Pull request by asimonson1125**
basically just sorting contributions by year so we can document who worked when.

### CODE REVIEW

**Comment by asimonson1125** [comment]
Anyone who works on devcade this year can add themselves to the json file in the 2023-2024 category

File: src/templates/home.html [hunk 1]
```diff
@@ -25,14 +26,14 @@ <h2>Who are we?</h2>
     </p>
     <p>
       The Devcade project is led by <a href="https://noahemke.com">Noah Emke</a>, 
-      a third year Game Design and Development major here at RIT. Noah leads a team of 
-      about a dozen CSH members, ranging from first years to fifth years. Many team members 
-      make their own games for the cabinet, while others work on the backend API and databases, 
-      some on the onboard software, and some on the physical design of the cabinet itself -- 
-      there really is something for everyone on this project.
+      a fourth year Game Design and Deveopment major at RIT. Noah leads a team of
```
Comment (wilsonmcdade) [inline src/templates/home.html]: Development

**Review by wilsonmcdade — APPROVED** [review]
Small spelling mistake otherwise lgtm

File: src/templates/home.html (hunk 1 above)
Comment (asimonson1125) [reply src/templates/home.html]: :fearful:

**Pull request merged**